source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -140,8 +140,11 @@ public PimsAcquisitionFile UpdateProperties(PimsAcquisitionFile acquisitionFile)
                 _acquisitionFilePropertyRepository.Delete(deletedProperty);
                 if (deletedProperty.Property.IsPropertyOfInterest.HasValue && deletedProperty.Property.IsPropertyOfInterest.Value)
                 {
-                    int propertyCount = _acquisitionFilePropertyRepository.GetAcquisitionFilePropertyRelatedCount(deletedProperty.PropertyId);
```

Comment:
if the GetResearchFilePropertyRelatedCount and acquisition sibling function are not used lets remove them